src/pfe/portal/modules/LoadRunner.js

```diff
@@ -462,12 +479,14 @@ module.exports = class LoadRunner {
           'Content-Type': 'application/json'
         }
       }
+      loadConfig.projectID = this.project.projectID;
```

Comment:
I think projectID is now the only parameter we need to send. See: https://github.com/eclipse/codewind/blob/7064533114f43f4621563f178c19817ba2e8a430/src/performance/server.js#L88
which is a big improvement as we only check the project id. (We may have been vulnerable to not being able to cancel a run if the config had been modified after it started before.)
You can just pass an object with:
`{projectID: this.project.projectID}`
to the http request and remove the need to pass `loadConfig` from this function and it's callers.